### PR TITLE
feat: Add swagger UI

### PR DESCRIPTION
**Pull request by patriciareinoso**
## Description

This PR adds a service for swagger UI docs.
This is an optional functionality, default behavior is not changed.

image

The swagger UI for the API documentation is automatically generated from annotations.

## How to use it:

To generate the swagger UI files run:
```
swag init -g backend/webui_service/swagger_ui_service.go --outputTypes go
```
The `docs.go` file will automatically be created in `webconsole/docs`

The swagger UI runs by default on `localhost`. If the webconsole server runs remotely, set the following environment variable.
```
export WEBUI_ENDPOINT=<webconsole-ip>:5000
```
Build the webconsole including the UI option:
```
make webconsole-ui
```
Access the swagger UI at:
```
http://<webconsole-ip>:5000/swagger/index.html
```
The `doc.json` file, which can be integrated in a frontend implementation, is available at:
```
http://<webconsole-ip>:5000/swagger/doc.json
```

## Documentation for code annotations

https://github.com/swaggo/swag

## Rationale
Swagger UI provides interactive API documentation where developers can see available endpoints, their parameters, and expected responses. They can also test endpoints directly from the documentation interface, making it easier to understand and use the API.

### CODE REVIEW

File: docs/docs.go
Comment (patriciareinoso) [inline docs/docs.go:1]: do we want to keep track of this file or ask for 
```
swag init -g backend/webui_service/swagger_ui_service.go --outputTypes go
```
before build?

File: backend/webui_service/swagger_ui_service.go [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+//	@contact.url	https://github.com/omec-project/webconsole
+
+//	@license.name	Apache 2.0
+//	@license.url	http://www.apache.org/licenses/LICENSE-2.0.html
+
+// @host		localhost:5000
+// @BasePath	/
+func AddSwaggerUiService(engine *gin.Engine) {
+    logger.WebUILog.Infoln("Adding Swagger UI service")
+    host := os.Getenv("SWAGGER_HOST")
```
Comment (patriciareinoso) [inline backend/webui_service/swagger_ui_service.go:31]: is `SWAGGER_HOST` a good name?

**Comment by thakurajayL** [comment]
Do you welcome comments in draft status?

File: backend/webui_service/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+make webconsole-ui
+```
+Access the swagger UI at:
+```
+http://<webconsole-ip>:5000/swagger/index.html
+```
+The `doc.json` file, which can be integrated in a frontend implementation, is available at:
+```
+http://<webconsole-ip>:5000/swagger/doc.json
+```
```
Comment (thakurajayL) [inline backend/webui_service/README.md:32]: Is this available only for developer or SD-Core user as well ? 
Do we ship the doc with regular container image?

Comment (patriciareinoso) [reply backend/webui_service/README.md:32]: This is available for developers and for sd-core users. 
At the moment, this functionality works only for the build using the ui tag (make webconsole-ui) but if we wish it is possible to make it the default behavior.
The swagger UI tool for go, automatically creates a web page that is served at `http://<webconsole-ip>:5000/swagger/index.html`. And it automatically creates the `doc.json` file and serves it.

File: configapi/api_default.go [hunk 3]
```diff
@@ -47,7 +53,15 @@ func GetDeviceGroups(c *gin.Context) {
 	c.JSON(http.StatusOK, deviceGroups)
 }
 
-// GetDeviceGroupsByName -
+// GetDeviceGroupsByName godoc
```
Comment (gatici) [inline configapi/api_default.go:56]:
```suggestion
// GetDeviceGroupByName godoc
```

File: configapi/api_sub_config.go [hunk 4]
```diff
@@ -295,7 +302,15 @@ func GetSubscribers(c *gin.Context) {
 	c.JSON(http.StatusOK, subsList)
 }
 
-// Get subscriber by IMSI(ueId))
+// GetSubscriber godoc
```
Comment (gatici) [inline configapi/api_sub_config.go:305]:
```suggestion
// GetSubscriberByID godoc
```

File: configapi/api_sub_config.go [hunk 5]
```diff
@@ -357,7 +372,15 @@ func GetSubscriberByID(c *gin.Context) {
 	c.JSON(http.StatusOK, subsData)
 }
 
-// Post subscriber by IMSI(ueId)
+// PostSubscriber godoc
```
Comment (gatici) [inline configapi/api_sub_config.go:375]:
```suggestion
// PostSubscriberByID godoc
```

File: configapi/api_sub_config.go [hunk 6]
```diff
@@ -449,7 +472,13 @@ func PatchSubscriberByID(c *gin.Context) {
 	logger.WebUILog.Infoln("Patch One Subscriber Data")
 }
 
-// Delete subscriber by IMSI(ueId)
+// DeleteSubscriber godoc
```
Comment (gatici) [inline configapi/api_sub_config.go:475]:
```suggestion
// DeleteSubscriberByID godoc
```

**Review by gatici — APPROVED** [review]
Looks Good.

File: configapi/api_default.go (hunk 3 above)
Comment (gatici) [reply configapi/api_default.go:56]: This is not a big deal and does not appear but we can keep it same with the method name.

**Comment by gab-arrobo** [comment]
@patriciareinoso please rebase your PR. Thanks!

**Review by thakurajayL — APPROVED** [review]

**Pull request merged**